h2/src/main/org/h2/engine/Database.java

```diff
@@ -1285,6 +1287,9 @@ public Role findRole(String roleName) {
      * @return the schema or null
      */
     public Schema findSchema(String schemaName) {
+        if (schemaName == null) {
+            throw DbException.get(ErrorCode.SCHEMA_NOT_FOUND_1, schemaName);
```

Comment:
This method returns `null` for unknown schemes, why an exception is required for a `null` name?

Reply:
you are absolutely right, my oversight